### PR TITLE
chore: add CHANGELOG.md

### PR DESCRIPTION
**Pull request by ableuler**
Add CHANGELOG.md to readme

### CODE REVIEW

**Comment by rokroskar** [comment]
The release was made before this was merged?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
+##  (2021-09-09)
+
+* chore: prep 0.10.1 release ([22fd987](https://github.com/SwissDataScienceCenter/renkulab-docker/commit/22fd987))
```
Comment (rokroskar) [inline CHANGELOG.md:3]: "chore" commits shouldn't be in the changelog most of the time

**Comment by ableuler** [comment]
recreated with `-p angular` option

**Comment by ableuler** [comment]
@rokroskar can we merge this?

**Review by rokroskar — APPROVED** [review]

**Pull request merged**